### DISCUSSION TITLE
迷你面积图最小高度是多少呢

### DISCUSSION THREAD

**Discussion by YY88Xu** [antvis/G2#6056] [Q&A] [answered]
### 问题描述

官网给的例子，迷你面积图的高度设置为 30px，面积图就不能正常绘制了
https://g2.antv.antgroup.com/zh/examples/general/mini/#area

### 重现步骤

import { Chart } from '@antv/g2';

const data = [
  264, 417, 438, 887, 309, 397, 550, 575, 563, 430, 525, 592, 492, 467, 513,
  546, 983, 340, 539, 243, 226, 192,
];

const chart = new Chart({
  container: 'container',
  width: 480,
  height: 30,  // 这里
});

chart.data(data);

chart
  .area()
  .encode('x', (_, idx) => idx)
  .encode('y', (d) => d)
  .encode('shape', 'smooth')
  .scale('y', { zero: true })
  .style('fill', 'linear-gradient(-90deg, white 0%, darkgreen 100%)')
  .style('fillOpacity', 0.6)
  .animate('enter', { type: 'fadeIn' })
  .axis(false);

chart.interaction('tooltip', {
  render: (e, { title, items }) => items[0].value,
});

chart.render();

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by hustcc** [comment, marked as answer]
把 padding margin 都设置为 0 吧，应该有一个默认 16px 的值。所以小于 32px 的时候，画布区域实际就位 0 了。

```ts
padding: 0,
margin: 0,
```
